### ISSUE TITLE
Goals, objectives and metrics for the bounty program

### ISSUE THREAD

**Issue by dckc** [rchain/bounties#783]
The 201805 Bounty Progress report (issue #759), had no statement of the objectives and evaluation criteria for the bounty program, which was noted as a deficiency during the board meeting Friday, June 15, 2018 (minutes not yet published). 

The purpose of this issue here is to establish Goals Objectives and Metrics for the bounty system.

Note draft initiated by @allancto :
 - [Interim Report; Goals, Objectives, and Metrics of the RChain Bounty System](https://docs.google.com/document/d/1jXm6Ks9OAe-0mgVOT2QtfaVkHndnOodj3uE5FJJXBNA)

### Benefit to RChain

Clarity for the CONTIBUTING.md, our **bounty-contract**.

### Budget and Objective

_Estimated Budget of Task:_ $1200 
_Estimated Timeline Required to Complete the Task:_ two weeks (target completion July 4)
_How will we measure completion?_ endorsement of metrics by RChain president, executive committee, Task Approval Committee, and/or RChain Board of Directors

cc @deannald @patrick727 @kennyrowe @lapin7 @leithaus 

### Legal

_Task Submitter shall not submit Tasks that will involve RHOC being transacted in any manner that (i) jeopardizes RHOC’s status as a software access token or other relevant and applicable description of the RHOC as an “asset”—not a security— or (2) violates, in any manner, applicable U.S. Securities laws._

**Comment by pmoorman** [comment]
Spoke with @kitblake just now, and he suggested we (him and me) maybe could help with this.

**Comment by dckc** [comment]
@lapin7 I just re-read your comment; which "we" needs to create metrics? Did Greg take the ball on that part too?

The first I heard of metrics for the bounty system was in the [May 9 weekly update][1] after board meeting. I'm still standing by for records of that board meeting too; I don't see them in https://github.com/rchain/board/tree/master/2018 .

[1]: https://github.com/rchain/bounties/wiki/Weekly-Debrief-Index#may-9th---community-debrief-76

**Comment by lapin7** [comment]
@dckc Yes getting the minutes correct and approved, is a cumbersome thing.

I guess it's up to the Task Approval Committee (@dckc @PatrickM727 @deannald) now to create metrics, evaluation criteria, objectives, KPI's  for the bounty system. I don't know if @leithaus comes up with objectives and criteria.

Until now the objective was: 

**Do whatever you can to get RChain on the rails, according to your own insights.**

Apparently this objective needs to be managed. 
The good thing is that the trust metric solves some of the problems with concern to:

- abuse of setting Budgets and Rewards
- setting priorities
- time and result management

**Comment by dckc** [comment]
> I guess it's up to the Task Approval Committee ... now to create metrics

No, I don't see anything about metrics in the description of the Task Approval Committee. Perhaps you're asking that we take this on? I'll consider it, but without records of the board meeting, we're missing a lot of essential context. Putting something together for the 201806 pay period seems impractical.

>  I don't know if @leithaus comes up with objectives and criteria.

What do you mean by that? You just wrote "Greg @leithaus will formulate the objectives and evaluation criteria of the bounty program" two days ago.

**Comment by deannald** [comment]
I agree with @dckc - I don't see anything about the Task Approval Committee taking on this body of work (creating metrics, etc.) nor direction on this from the Executive Committee minutes from this week.

**Comment by lapin7** [comment]
I meant to say: @leithaus said that he would formulate objectives and criteria, but I’m not sure that he will actually do it.

**Comment by allancto** [comment]
@kennyrowe can we have a rough summary of what the board discussion was on Jun15 (complete minutes would be great, but summary relative to bounty will do)? @lapin7 @deannald @pmoorman @patrick727 @dckc @kitblake are you all available at 9am PDT monday june 25 to discuss objectives and metrics? @leithaus (and others) you are invited certainly as well if it fits your schedule. @lapin7 will you set up the zoom?

**Comment by deannald** [comment]
@allancto I would prefer to get some clarity and direction on who will be working on objectives, criteria and metrics before we start working on them with the group you mention above.

**Comment by dckc** [comment]
I'm not available during business hours Monday. Not until Friday.

**Comment by dckc** [comment]
@lapin7 I just re-read your [Jun 19 comment][1]:
> The 201804 and 201805 Bounty Progress Report was received, but the objectives and evaluation criteria were missing. 

In what way were they missing? Why did anyone expect them in that report? That's the first mention of objectives and evaluation criteria in issue #678.

[1]: https://github.com/rchain/bounties/issues/678#issuecomment-398328976

**Comment by lapin7** [comment]
@dckc 
@leithaus expected them in the report.

**Comment by allancto** [comment]
@deannald based on your comment above I changed the title of this issue to "Interim Report". @dckc , @lapin7 I have also edited the task description to include discussion from the comments above.

**Comment by dckc** [comment]
@ian-bloom thinks he can ping @leithaus about this in the next day or so

**Comment by dckc** [comment]
@allancto I ran into this issue in another context and was a little surprised to see the "interim report:" prefix in the summary. I remember now... and that would make sense, if github would let us change the author to you (a feature I'm used to from [Trac](https://trac.edgewall.org/)). But I'm the author, so I tweaked it to something in between. Input to the TAC is not my goal. My goal is to get the whole job done, whatever it is.

**Comment by deannald** [comment]
@ian-bloom - any luck in following up with @leithaus on this issue on Friday?

**Comment by kitblake** [comment]
@deannald Ian doesn't read his Github mail. He told me to use normal mail or Discord. 

@dckc Particularly for this issue, it would be useful to have the minutes of the Board meeting. 
"...establish Goals Objectives and Metrics for the bounty system" is a huge scope.

**Comment by deannald** [comment]
@kitblake - I also emailed him as well.

**Comment by dckc** [comment]
recall from our July 4 meeting (#823):
> @allan - ... **Will write objectives and metrics based on the stated purpose**, though I don’t think we’re meeting them.
> @dckc: if you write objectives that we’re clearly not meeting, that will advance the discussion.

What news, @allancto ?

**Comment by dckc** [comment]
@allancto I see you made substantial progress in objectives and metrics sections in [the report][1] as of July 20. IOU a review.

[1]: https://docs.google.com/document/d/1jXm6Ks9OAe-0mgVOT2QtfaVkHndnOodj3uE5FJJXBNA/edit#heading=h.uvy2o57ie9wt

**Comment by deannald** [comment]
@allancto - Me too (owe you a review). I'm out the next 2 days, so probably some time Thursday...

**Comment by allancto** [comment]
I am closing this issue subject to the measure of completion listed in the issue statement  (endorsement of metrics by RChain president, executive committee, Task Approval Committee, and/or RChain Board of Directors). 

Budget requested: 3600 
Valuation model: Value should be assessed by TAC and Directors (reflecting value to our Cooperative). 
%contribution: 
82% @allancto 
6% @pmoorman 
6% @lapin7 
3% @jimscarver 
3% @dckc

**Comment by dckc** [comment]
I don't see the endorsements

**Comment by allancto** [comment]
@dckc thanks. I can't tell from your response whether the TAC will be endorsing it or not. I'll be happy to work to get it endorsed by our board or president if that's your request, but in that case I'd like to mark the issue as ONGOING and get the part that's already completed voted. I've been working on it for the last 6 weeks. 

Thanks! -@allancto

**Comment by dckc** [comment]
The goal here is to come up with something the TAC endorses. Whether we endorse the current draft isn't clear; I haven't finished my review.

It's reasonable to request a reward for a small part of the budget for getting past a blank page, but what looks like the last 20% of the document is often 80% of the work, so most of the budget should be reserved for when the whole job is done.

**Comment by dckc** [comment]
I changed the estimated duration from two weeks to six months based on the amount of time it's taking to get feedback from the parties involved.

**Comment by kitblake** [comment]
Given the current examination of the bounty system, this doc, with its sections on metrics and historical achievements, has turned out to be quite timely. Initially I thought it was an 'extra' but it gathers and provides a lot data to work with. 

Note that the title includes "Interim Report". Once things take a definitive form, we might bring out an update based on followup and evolution of the approach. Some of the content can also be used for future communication about our innovative and mesh-based bounties solution.

**Comment by dckc** [comment]
@allancto what I see in [the current draft][1] is a bit different from what I expected, but I suppose it's OK (with one editorial exception).

In our July 4 discussion, I said "if you write objectives that we’re clearly not meeting, that will advance the discussion". So I expected not only metrics ("number of burgers sold") but also targets ("at least 100 burgers per month") so that we could tell whether we were meeting them or not.

I don't see the targets. But I guess I don't need them.

I'm willing to organize reporting quarterly on the metrics in the current draft: number of participants, utilization of communication channels, etc. (To be clear: I'm also content with the level of reporting we are already doing, so I would be OK if this metrics draft were set aside.)

I reviewed the introductory material only lightly. I was going to suggest removing it, but I see it's valuable to at least one reviewer (@kitblake ) so I'm not going to suggest that after all.

----

The one editorial exception regards "all other human creations, are inevitably created with “bugs”. See comment in the doc.

Also: I don't have access to the version history, so I can't tell when it was last revised.

[1]: https://docs.google.com/document/d/1jXm6Ks9OAe-0mgVOT2QtfaVkHndnOodj3uE5FJJXBNA/edit#

**Comment by jimscarver** [comment]
I voted on the above proposal and suggest we show our sentiment with 👍 👎 👉 etc.

**Comment by jimscarver** [comment]
Greg suggests we find sponsors in the coop and the partners for bounty activities and guide members to those activities.  

Work on wallets, dApps, training, testing, finding, reporting, creating tests and finding the bugs are parimont activities that should comprise a significant part of our budget. 

The social ledger and decision tools to be put on rchain when possible was also a significant ask from many at rcon3 in the realms of organized self organization and decision making.

Marketing is a bit of a pickle, Community building and marketing overlap. We need a clear mandate from marketing on what to guide members to do but not preclude other efforts of value.

Greg's view is that we need to guide the system toward "Sponsored" major want's of the coop core members but not preclude other efforts that have significant member support.

**Comment by dckc** [comment]
All guides, please note [my proposal to the TAC][p] above to reboot all labels. You're more than welcome to show your sentiment with :+1: and :-1: or add comments that go beyond just yes / no.

[p]: https://github.com/rchain/bounties/issues/783#issuecomment-420802890

This list from #925 looks like a handy way to notify all of you:

 - [ ] Operations: @TrenchFloat, @jimscarver
 - [ ] Marketing: @pmoorman @AyAyRon-P @kitblake
   - separate budget: #817
 - [ ] Translation: @AyAyRon-P, @pavlos1851
   - separate budget: #869
 - [ ] Governance: @jimscarver, @barneycinnamon, @rayzor
 - [ ] Development: @dckc, @Jake-Gillberg 
 - [ ] member-site: @PatrickM727 @andrekuipers @kitblake @ian-bloom
 - [ ] Community Building: @PatrickM727, @allancto, @Ojimadu 
 - [ ] RChain Technical Literacy: @JoshOrndorff @David405
 - [ ] China: @zsluedem
 - [ ] Education: @TrenchFloat
 - [ ] Greeter: @makys @ian-bloom
 - [ ] Events: @Ojimadu
 - [ ] Voting: @Jake-Gillberg @jimscarver @allancto
 - [ ] Branding: @pmoorman @AyAyRon-P @kitblake
 - [ ] Africa: @David405 
 - [ ] Social media: @azazime @jasoncruzzy @Ojimadu

**Comment by dckc** [comment]
Some other coop memebers have been discussing related options, and several of them joined a Zoom session that I just hosted. Unfortunately I didn't capture the list of attendees. I remember @ddayan and @JoshOrndorff and @luigidemeo were there.

After some clarification, it seemed that everybody's goals are largely consistent with the proposal above, with an emphasis on the sponsorship aspect that @jimscarver mentioned.

We did some brainstorming around the challenge of @PatrickM727 's availability to directly participate in the TAC.

I agreed to add Ned from the solutions group to the Development sponsors. And @JoshOrndorff expressed interest in guiding in this area. (Perhaps I / we should split Development into Core bounties and Tools bounties?)

**Comment by ysgjay** [comment]
@dckc Please add me to "Greeter". Amaka (makys) is no longer a greeter

**Comment by JoshOrndorff** [comment]
I think everyone on the zoom call agreed that @dckc 's proposal to remove all label guides and add them back only as necessary and fruitful along with  @jimscarver suggestion for sponsoring were good and necessary first steps. We also kicked around next steps but everyone was prepared to take this first step.

**Comment by TrenchFloat** [comment]
Sounds good to me, provided we do more than what has been done to notify _everyone_ being rewarded now. I can hear the "What? My certification is gone?!" already :)

**Comment by dckc** [comment]
If each of us notifies those we have certified, we should be all set.

**Comment by AyAyRon-P** [comment]
@dckc is there another meeting slated any time soon? I'd like to continue working in/for/with the same labels - Translation and Marketing to help manage RAM expectations, issues requests and deliverables.

**Comment by allancto** [comment]
I confirm what @JoshOrndorff said, there is consensus to reboot. 

> I think everyone on the zoom call agreed that @dckc 's proposal to remove all label guides and add them back only as necessary and fruitful along with @jimscarver suggestion for sponsoring were good and necessary first steps. We also kicked around next steps but everyone was prepared to take this first step.

An important part of the first step needs to be a clearer understanding and possibly of what the role of label guides to actually be. We've gained a lot of experience in our 3 months of trialing. I'm linking a document intended for contributions on how to incorporate and evolve the system, including Purpose, Labels, and Mechanism. 

[Label guides system review 2018_09](https://docs.google.com/document/d/1Amk--A2fNMllAVSDFmtkY0j_L_7u8-Clt9hpIghJ_8U/edit?usp=sharing). Please take a look and add your viewpoints if you are interested in shaping this reboot (permissions are set to "comment" and I will include all suggestions with authorship attribution, thanks @dckc for emphasizing the importance of attribution). 

Thanks! -@allancto

**Comment by dckc** [comment]
@deannald and I met, along with @JoshOrndorff and @ddayan in behalf of @PatrickM727 to review the plan above. We plan to go ahead, effective Weds Sep 19, by deleting all trust certs and only adding them back as we have sponsorship etc. in place.

We also spent a little time working through [overdue task proposals](https://rewards.rchain.coop/index.php?-table=task_approval_overdue) and brainstorming about improved documentation.

**Comment by JoshOrndorff** [comment]
I confirm that Dan's post is correct and reflects my intentions


-----BEGIN PGP MESSAGE-----

owE1kX1QVFUYxpdlneCGoA0ZgsJBRhg+3FWI74QtlgUVRSVALKKze8/uXrj3nuu9
l2WvSRhj05c2m4mAg1O6GPsHiaOEipOUAypfk2Y1yRhCjV/oaNKYCqx2lvT8dc68
z3ne3/u8zrm+Kj8fY8lY5d4LP2X4fOvTb5pvwtW8rHBIkqAVaWWH/FbLnIlVwIx5
CyNyQLZBGRggHyMBAUsyYCRSEkVklgHkaSAiC0vuEuAUwPAy4mUG8xJFLfMeitLT
CPI8ZOlZ8SrAITkBQBbzVlDDyDagX40lW6HI01i0WGY1epqGCuSJGTAhG2QtAFuA
fj2URcZctTY1MRXImHS1M6iGsCEgsEQMTdiOtKD02ZMorBhAG4J0AkAWCwFk7IiU
aQkUIQGsSE8AJgXQiEUyQ1AgywJZrCbTmZFIZvFyYJ5VAKRpb5304YAJmqsAlEAN
AjZI3CSBDIpFycYIAMlmrZeYdDcjLUUREMhKmGhIIAAClpFlFgGZ4RCowWLV/6Yi
rrbaACEX6WpShFIVEERMUiZ/ZxlMImR4ScYiN0tJNkXy54jGjmhAY3M1R+yhN3Et
9Ym6QKPy8VOlLIjUdOhnNhj7O4tqbRvnP1/2HLV3ryGLIE3M9JUkdgU/y11rxpyK
8p/3XLp9fJ7qa7nvafGnL8X5fx4XviHkyaC9KfxK6SmAnhjbLQuPLYhOyVP3tV3T
TeR11z/afFM/kKbL7GjbSh0a528XGvYcTNjtXP5z113X9NX9O1VNjexl2Ke69+bc
YNf2c3Toke+Vi9H8IcNXybGhsDss8ztw6R3/AyX1j0/t2jXRdf2DtcEdsa8wA/dz
OyfW63wd7XHuHR4Hm14b/1eU8/TLdXsKRr/5Y9G2ygfvOV0t0syUGxSXbixxSjPM
w2hTmidmr6s5Z5PuRPOPK9cNHryJdILreJqmtyHVc6fs9VjfwvMrpsuPClfCR+6+
+pmh+/5Z6jVPxZnA8/1pa5JC8gc2Hzk59MZF/UfH2DT1rdaK/Dq/wICkLMOXF2Ia
Lm+JO8lYs23CpFvK+fhx/tJWYTLwV8e2F82rDbF+yeEOx9SNiNrgO7eibsj/cAsH
xSXHh3VZ61ojl2T8vS8qvvza6BD1hfvP+OKR/dFjV90zQeXLcgMac8evtx8FxrDD
DZ7hnkfG0KDUmea83SszFju3/pBk7vowZTLyYdCmlE5rmRjWqwQo41lRkZqu0SZ3
U73r34zsqNMRdcr7TTvf/n1xfe+J226txpEz5OEKD797tqdx4F52YUSueiR0aqmS
nph/ZnA480DoA/8dyxNxcttY1S/TY63qnhz7uZai9LJL+7YkCpoX1vgXeH6rqHz6
Hw==
=M/XA
-----END PGP MESSAGE-----

**Comment by TrenchFloat** [comment]
I added to Allan's [system review doc](https://docs.google.com/document/d/1Amk--A2fNMllAVSDFmtkY0j_L_7u8-Clt9hpIghJ_8U/edit#) a section on the proposal from [9/12's RAM meeting](https://github.com/rchain/bounties/issues/958) to require guides to sync up with coop "higher ups" for their label and talk about it in issues like #925.

> To ensure accountability from the label guides that they are regularly aligning bounty system work with coop goals, it has been proposed (9/12 RAM meeting) that a guide from each label post in each “review of X pay period by guides” issue (like #925 for August) a summary or link of their communications with their relevant connection outside the bounty system. “Relevant connections” for each guide are to be determined, but inferred examples would include Medha P. for the Development label, Patrick M. for Marketing, etc.

(Maybe there's a better word than "higher ups". I just mean people that guides would want to talk to to sync up on goals.)

**Comment by allancto** [comment]
@TrenchFloat yes, that's the direction we should be going. Furthermore in this issue two fundamentally different things seem to be conflated: is this a reboot of label guides or of trustmetric certification? Probably both require a reboot, but the discussions are (imo) totally different. 

What are the qualifications for Trustmetric certification? Trustworthiness, understanding of the voting system, understanding of what makes budgets realistic and SMART. What are the qualifications for Label guiding? Connection within our Cooperative, domain specific knowledge of area or "department". Both are important.

**Comment by JoshOrndorff** [comment]
@allalcto The proposal is that both trust metric and label guides be reset.

**Comment by allancto** [comment]
@JoshOrndorff yes, dual reboot. The question is the mechanism for each. Once the trustmetric is reseeded there should be certification voting. Is there guidance as to how that will best proceed? Once label guides are rebooted we'll need to make sure we have a good plan for Labels that represent Cooperative activities and requirements, and find contributors to fill those roles. I've seen @barneycinnamon  and @ysgjay (and I'm also hoping for @luigidemeo , @ddayan , @Viraculous, yourself and perhaps others to come forward). What's an appropriate venue for discussing requirements and qualifications and volunteers for guides? What's an appropriate venue for discussing requirements and guidelines for voting Trustmetric ratings? 

Thanks! -@allancto

**Comment by dckc** [comment]
@allancto writes:
> two fundamentally different things seem to be conflated: is this a reboot of label guides or of trustmetric certification?

I don't see it as two different things.

> Once the trustmetric is reseeded

reseeded? The seed remains the same: the TAC (@PatrickM727 , @deannald , and myself).

> there should be certification voting. Is there guidance as to how that will best proceed?

Yes: https://github.com/rchain/bounties/issues/783#issuecomment-420802890

The requirements for getting certified to vote / guide an area are

> making the relevant sociocratic connections, getting mandate, establishing budget criteria, and making projections.

So write all that up for any area you're interested to see operational and if it looks good, the TAC can certify people who agree to that "charter" for the area label. I gave a nearly complete example for Development above.

**Comment by David405** [comment]
@dckc @joshOrndorff @jimscarver I am yet to wrap my head around the proposal.

By sponsorship, do you mean support for a work in the bounty, certification for rating or endorsement to become a label guide?

**Comment by dckc** [comment]
The most direct form of sponsorship is for bounty rewards to come directly out of the budget of some other part of the coop, such as Marketing or the solutions group.

I can imagine other forms. Perhaps the Executive Committee would agree that it's worth allocating up to  $X/quarter for work in some area.

In general, I guess sponsorship is when someone in a position to speak for the coop as a whole agrees how a certain amount of money should be spent through the bounty system.

I expect this question is easier to deal with in the concrete than in the abstract. Is there a specific area that you are asking about?

The new labels don't need to be the same size and shape as the old labels, but just using them as an example... I talked about **Development** already...

I would expect to see some sponsorship from Marketing for anything like these labels: **Branding**, **Community Building**, **Events**, **Marketing** (duh), **Social media**, **Translation**.

The Executive Committee used to be interested in **member-site** work, but we haven't talked about it for quite some time.

The **RChain Technical Literacy** / **Education** areas seem important, but I don't have a clear sense of what part of the coop deals with them. Perhaps the solutions group is interested to sponsor work in this area, @JoshOrndorff ?

**Governance** is a puzzler. We say right on the homepage "Cooperative governance and a sustainable economic model make RChain a public utility which can be applied to the greatest problems today." and yet I don't know of anybody leading work on Governance. **Voting** work has petered out somewhat.

Regarding **China**, I think I saw @ottermagically say the RChain Asia group is likely to organize something separate from this bounty system. I don't know if there is any staff or the like allocated to Africa.

I asked around about **Operations**; all I learned is that @allancto is covering some of the duties of @lapin7 .

**Greeter** work seems to have migrated out of the bounty system to the staff. @ysgjay can you help us understand how Greeter work is compensated, if it is?

**Comment by ysgjay** [comment]
@dckc for Greeter work, I am currently not compensated (Have been a greeter since April). I'll probably aim for an SOW since it's a consistent position.

**Comment by dckc** [comment]
## Reboot under way

> We plan to go ahead, effective Weds Sep 19, by deleting all trust certs and only adding them back as we have sponsorship etc. in place.

Response to the proposal has been positive. I haven't see any reason not to proceed, so yestereday I began renaming **member-site** to **zz-member-site** and likewise other area labels. I changed their color to some hue of red.

Someone please review PR #973 to update ISSUE_TEMPLATE. I intend to revise [Bounty Task Guides](https://github.com/rchain/bounties/wiki/Bounty-Task-Guides).

I just now reset the trust ratings:

```sql
delete from trust_cert;
-- 404 rows processed
```

There was a bug I had to fix to recompute ratings:

* a097805       2018-09-20      handle 0 trust certs

Note the [archive of trusted authorities](https://gist.github.com/dckc/dab60698c215bdff16008c1e150f48ac) from Sep 9 in #925.

**Comment by dckc** [comment]
Yes, it's important to help each other find sponsors. I am making an effort to do so; for example: https://github.com/rchain/bounties/issues/974#issuecomment-424515275

**Comment by dckc** [comment]
In #982, @azazime writes:
> Is the Bounty Program still a decentralized system and how does these new reforms support it?

I stipulate that the reboot is a regression w.r.t. decentralization. I look forward to doing better.

But to understand why I think this is a reasonable step to take, please see [Urbit isn't even really decentralized! It has a government!](https://urbit.org/posts/objections/#gov); an excerpt follows:

> TLDR: it's technically impossible for Urbit to be decentralized at this stage of its life. Urbit is designed to achieve decentralization, not be born with it.
>
> A young network can't afford to be decentralized. It has to act as a unit. Because it has no network effect, it needs every scrap of efficiency it can get.
> 
> A decentralized general-purpose computing network like Urbit can only be bootstrapped by a central government. For example, by default your urbit upgrades its OS automatically (like an "evergreen" browser) with signed network updates. Whoever signs these updates is a government by definition.
> 
> When it's impossible to eliminate centralized power, it becomes necessary to tame, stabilize, and limit it. This should not be surprising to anyone living under a constitutional government.
> 
> As we found out with the DAO, if a network isn't ready to be decentralized, its developers shouldn't even try. If Ethereum is hard-forked to roll back the DAO (which looks like the most probable result), it acknowledges that it both has a central government, and needs a central government. Decentralization is good. Pretend decentralization -- "decentralization theater" -- is harmful, and not a effective path to actual decentralization.

**Comment by dckc** [comment]
in #982 @azazime writes:
> How effective are the new reforms in the bounty system?

I you refer to the Sep 19 reboot that is the subject of this issue, so I recommend keeping discussion of the effectiveness of the reboot together here with the other discussion of the reboot.

**Comment by dckc** [comment]
@kovmargo writes in #982:

> - Will [we] continue to vote for the issues that we open/close in September?

You should, yes. Everyone has always been encouraged to vote, regardless of their trust ranking. Voting wisely establishes a record that's useful when considering trust certification.

> - From which date for open-issues sponsors will be needed? For example, issues that were open after 19 Sep should have sponsors

Starting with the 201809 pay period, the TAC intends to delegate impact on budgets and rewards only to voters who agree to more clear sponsorship and budget norms, regardless of when the issue was opened.

> - There are 9 people with the rating at the rewards not 3 as it were describe above. Why? What is their role now? Is there a connection with sponsors?

1. @PatrickM727 - appointed to the TAC by the executive committee 2018/06/19
1. @dckc - appointed to the TAC
1. @deannald - appointed to the TAC
1. @MParlikar - sponsor for **core-dev** per Sep 28 discussion in #273
1. @AbnerZheng - agreed to **core-dev** budget norms in #273
1. @ddayan - agreed to **core-dev** norms in #273
1. @JoshOrndorff - **developer-education** sponsor per Sep 29 discussion in #692
1. @ottermagically - certified by @dckc to deal with **China** bounties, as discussed in #934 (most likely: to use a separate system for them)
1. @zsluedem - likewise re **China**

> Is there a list of all departments and the people who head them and those who can sponsor? I will explain. For example, I want to do smth valuable for coop and first I need to find the sponsor. What should be my steps?


The list of departments is https://github.com/rchain/bounties/labels along with https://github.com/rchain/bounties/wiki/Bounty-Task-Guides

Only **core-dev** and **developer-education** have sponsors established at this time.

If you have valuable work in other areas, you and I and the rest of us share the burden of finding sponsors. As I mentioned above, it's easier to deal with such things in specific cases rather than in a general way. So your first steps should be to share what valuable work you want to do, perhaps by opening an issue.

> @dckc "As explained in #783, it's important to help each other find sponsors for good work. I gave an example of how I am making an effort to do so."
Sorry, I read all #783 and didn't find clear steps how to find sponsors.

In my Sep 26 comment above, I referred to a Sep 25 recommendation:

>  "@allancto This seems to overlap with responsibilities of Derek Beres, Director of Content. I recommend you discuss the benefit to RChain with him." -- https://github.com/rchain/bounties/issues/974#issuecomment-424515275

It was clear enough for @allancto . Again, what good work do you have in mind?
